src/hardware/vga_draw.cpp

```diff
@@ -1614,23 +1614,27 @@ void VGA_SetupDrawing(uint32_t /*val*/)
 	case M_HERC_GFX:
 		vga.draw.blocks=width*2;
 		width*=16;
-		aspect_ratio = (static_cast<float>(width) / static_cast<float>(height)) * (3.0f / 4.0f);
+		assert(width > 0);
+		assert(height > 0);
+		aspect_ratio = static_cast<float>(width * 3) / static_cast<float>(height * 4);
 		VGA_DrawLine = VGA_Draw_1BPP_Line;
 		break;
 	case M_TANDY2:
 		aspect_ratio = 1.2f;
 		doubleheight=true;
-		if (machine==MCH_PCJR) doublewidth=(vga.tandy.gfx_control & 0x8)==0x00;
-		else doublewidth=(vga.tandy.mode_control & 0x10)==0;
+		doublewidth = machine == MCH_PCJR
+		                      ? !(vga.tandy.gfx_control & 0x8)
+		                      : !(vga.tandy.mode_control & 0x10);
```

Comment:
From formatting POV I don't like this change - it's even less readable than the old version - `?:` is way too complicated in here.

Please move back to old version of this code (including comparison to 0x0), just fix formatting in these two lines.